pkg/generator/gen_version_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generator
+
+import (
+	"bytes"
+	"testing"
+)
+
+const versionExp = `package version
+
+var (
+	Version = "0.9.2+git"
+	GitSHA  = "Not provided (use ./build instead of go build)"
```

Comment:
@theishshah It doesn't make sense to have this right now. Can you remove the GitSHA variable altogether in a followup PR.
This is only needed when you set the GitSHA env at build time.
https://github.com/coreos/etcd-operator/blob/master/hack/build/operator/build#L14-L22

Which we don't in the SDK build script
https://github.com/operator-framework/operator-sdk-samples/blob/master/vault-operator/tmp/build/build.sh#L18

Reply:
Will do, and will tag that PR here

Reply:
PR #306